### ISSUE TITLE
【3.2 线性回归的从零开始实现】错误内容

### ISSUE THREAD

**Issue by youngxhui** [TrickyGo/Dive-into-DL-TensorFlow2.0#15]
# 代码输出

将原有的 `features[0], labels[0]` 代码应该贴在下面代码区。

![image](https://user-images.githubusercontent.com/16971804/70145012-189f5c00-16da-11ea-87bb-9ea98e06b976.png)

# 未修改内容 

1. 仍旧是 pytorch 内容。

![image](https://user-images.githubusercontent.com/16971804/70145095-4c7a8180-16da-11ea-8645-2b63c6cab098.png)

2. 应该为 `matmul` 函数

![image](https://user-images.githubusercontent.com/16971804/70145161-6c11aa00-16da-11ea-9ca5-1d997e9d8786.png)

**Comment by SwordFaith** [comment]
谢谢提醒，正在处理
